fido2/nfc.py

```diff
@@ -0,0 +1,113 @@
+
+from .ctap import CtapDevice
+from .hid import CAPABILITY, CTAPHID
+from .pcsc import PCSCDevice
+from smartcard.Exceptions import CardConnectionException
+
+
+class CtapNfcDevice(CtapDevice):
+    """
+    CtapDevice implementation using the pcsc NFC transport.
+
+    :cvar descriptor: Device descriptor.
+    """
+
+    def __init__(self, descriptor, dev):
+        self.descriptor = descriptor
+        self._dev = dev
+
+        # init card
+        self._dev.connect()
+        self._ats = self._dev.get_ats()
+        if (self._ats is None) or (len(self._ats) == 0):
+            raise Exception('No ATS')
+
+        self._app_select_result, sw1, sw2 = self._dev.select_applet()
+        if (self._app_select_result is None) or \
+           (len(self._app_select_result) == 0) or \
+           (sw1 != 0x90):
+            raise Exception('Select error')
+
+        return
+
+    def get_pcsc_device(self):
+        return self._dev
+
+    def __repr__(self):
+        return 'CtapNfcDevice(%s)' % self.descriptor
+
+    @property
+    def version(self):
+        """CTAP NFC protocol version.
+
+        :rtype: Tuple[int, int, int]
+        """
+        return [1, 0, 0]
```

Comment:
This should match the docstring and return a tuple.

Reply:
not a tuple and not a string(
current hid.py returns only one int

what to do? 
1. make as current, fix docstrings
2. make tuple[1,2,3] and fix hid.py?

i prefer 1st)

Reply:
Fixed 1st way